src/test/java/org/folio/innreach/controller/InnReachTransactionControllerTest.java

```diff
@@ -217,29 +216,33 @@ class InnReachTransactionControllerTest extends BaseControllerTest {
   @Autowired
   private RequestPreferenceService requestPreferenceService;
 
-  @MockBean
+  @MockitoBean
   private InventoryClient inventoryClient;
-  @MockBean
+  @MockitoBean
   private CirculationClient circulationClient;
-  @MockBean
+  @MockitoBean
   private UsersClient usersClient;
-  @MockBean
+  @MockitoBean
   private InnReachClient innReachClient;
-  @MockBean
+  @MockitoBean
   private HoldingsStorageClient holdingsStorageClient;
-  @MockBean
+  @MockitoBean
   private RequestPreferenceStorageClient requestPreferenceClient;
-  @MockBean
+  @MockitoBean
   private ServicePointsClient servicePointsClient;
-  @MockBean
+  @MockitoBean
   private ServicePointsUsersClient servicePointsUsersClient;
-  @SpyBean
+  @MockitoSpyBean
   private RequestService requestService;
-  @SpyBean
+  @MockitoSpyBean
   private InnReachTransactionActionNotifier actionNotifier;
 
   private static final HttpHeaders headers = circHeaders();
 
+  RandomStringGenerator generator = new RandomStringGenerator.Builder()
+    .withinRange('a', 'z')  // Define character range
+    .build();
```

Comment:
`build()` is deprecated: https://github.com/apache/commons-text/blob/rel/commons-text-1.13.0/src/main/java/org/apache/commons/text/RandomStringGenerator.java#L134
```suggestion
    .get();
```